example/lib/view/home.dart

```diff
@@ -0,0 +1,92 @@
+import '../utils/dimension_constants.dart';
+import '../utils/string_constants.dart';
+import '../route/route_constants.dart';
+import 'package:flutter/material.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
```

Comment:
Think this, what happen if you instead of 5 widget you have 40, if i follow this logic i need to create 40 ElevatedButtons

Reply:
Try to create a class that has as parameters a string for the route and a string for the name of the widget, then you can create a private list like this:
routeList = [('Title Widget', RouteConstants.widgetTitle) ...] (I define a element of the list)

Reply:
An then you only need to create a ListBuilder with that list, instead to create 40 Elevated Button individually, you only need to define one

Reply:
Thank you for this approach, you are clearly right. It would be absurd to duplicate so many code

Reply:
Fixed. A list of widgets was created that will be displayed in just one ElevatedButton